docs/gettingstarted/index.md

```diff
@@ -6,18 +6,19 @@ This is a simple guide to get you from nothing to a basic mod. The rest of this
 From Zero to Modding
 --------------------
 
-1. Obtain the Mod Development Kit (MDK) from forge's [files][] site. (Look for the MDK file type)
-2. Extract the downloaded MDK to an empty directory. You should see a bunch of files, and an example mod is placed in `src/main/java` for you to look at. Only a few of these files are strictly necessary for mod development, and you may reuse these files for all your projects These files are:
+1. Obtain a Java 8 Development Kit (JDK) and a 64-bit Java Virtual Machine (JVM). Minecraft and MinecraftForge both compile against Java 8 and as such should be used for development. Using a 32-bit JVM will result in some problems when running the below gradle tasks. You can obtain one from [AdoptOpenJDK][jdk].
+2. Obtain the Mod Development Kit (MDK) from forge's [files][] site. (Look for the MDK file type)
+3. Extract the downloaded MDK to an empty directory. You should see a bunch of files, and an example mod is placed in `src/main/java` for you to look at. Only a few of these files are strictly necessary for mod development, and you may reuse these files for all your projects These files are:
     * `build.gradle`
     * `gradlew.bat`
     * `gradlew`
     * the `gradle` folder
-3. Move the files listed above to a new folder, this will be your mod project folder.
-4. Choose your IDE:
+4. Move the files listed above to a new folder, this will be your mod project folder.
+5. Choose your IDE:
     * Forge only explicitly supports developing with Eclipse, but there are additional run tasks for IntelliJ or Visual Studio Code environments. However, any environment, from Netbeans to vim/emacs, can be made to work.
     * For both Intellij IDEA and Eclipse, their Gradle integration will handle the rest of the initial workspace setup. This includes downloading packages from Mojang, MinecraftForge, and a few other software sharing sites. For VSCode, the 'Gradle Tasks' plugin can be used to handle the initial workspace setup.
     * For most, if not all, changes to the build.gradle file to take effect Gradle will need to be invoked to re-evaluate the project, this can be done through Refresh buttons in the Gradle panels of both the previously mentioned IDEs.
-5. Generating IDE Launch/Run Configurations:
+6. Generating IDE Launch/Run Configurations:
     * For Eclipse, run the `genEclipseRuns` gradle task (`gradlew genEclipseRuns`). This will generate the Launch Configurations and download any required assets for the game to run. After this has finished refresh your project.
     * For IntelliJ, run the `genIntellijRuns` gradle task (`gradlew genIntellijRuns`). This will generate the Run Configurations and download any required assets for the game to run. After this has finished edit your Configurations to fix the "module not specified" error by changing selecting your "main" module.
```

Comment:
```suggestion
    * For IntelliJ, run the `genIntellijRuns` gradle task (`gradlew genIntellijRuns`). This will generate the Run Configurations and download any required assets for the game to run. After this has finished, edit your configurations to fix the "module not specified" error by changing selecting your "main" module.
```
The 'fixing the error' part needs to be verified if it still exists, and if possible an appropriate note placed if it is an IntelliJ bug or a ForgeGradle bug.

Reply:
I don't think I've seen it the one or two times I've used IntelliJ, but that is also not the main IDE I use so I could be mistaken.

Reply:
I use IntelliJ, haven't seen that error

Reply:
Probably is an artifact from either FG2 or IntelliJ 2018- then. I'll remove it.